AndroidAnnotations/androidannotations-otto/otto/src/main/java/org/androidannotations/otto/handler/AbstractOttoHandler.java

```diff
@@ -64,6 +64,10 @@ public void validate(Element element, ElementValidation valid) {
 
 	@Override
 	public void process(Element element, EComponentHolder holder) throws Exception {
+		if (!annotationHelper.enclosingElementHasEnhancedComponentAnnotation(element)) {
+			// do nothing when otto annotations are used in non-enhanced classes
+			return;
+		}
```

Comment:
this is a empty line? :D maybe because i already removed the check you were talking about? :)